code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -27,7 +27,7 @@
 	return
 
 
-/obj/item/reagent_containers/pill/attack(mob/M, mob/user, def_zone)
+/obj/item/reagent_containers/pill/attack(mob/M, mob/user, obj/item/bodypart/affecting)
```

Comment:
Unrelated changes

Reply:
Related to patches caring about which body part you're targeting to do their effects.